### PR TITLE
feat: tutorial 12.4

### PR DESCRIPTION
**Pull request by Anisha-Alexander**

### CODE REVIEW

File: Tutorial12.4/example-1/menu.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>The Net Ninja - CSS Flexbox</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+  </head>
+  <body>
+    <div class="wrapper">
+
```
Comment (donjude-eng) [inline Tutorial12.4/example-1/menu.html:9]: @Anisha-Alexander Install prettier to format the code?

File: Tutorial12.4/example-3/grid-vs-stack.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        </article>
+        <article>
+          <h2>What's a horse doing on a spaceship...?</h2>
+          <p>Etiam aliquam sem ac velit feugiat elementum. Nunc eu elit velit, nec vestibulum nibh. Curabitur ultrices, diam non ullamcorper blandit, nunc lacus ornare nisi, egestas rutrum magna est id nunc. Pellentesque imperdiet malesuada quam, et rhoncus eros auctor eu. Nullam vehicula metus ac lacus rutrum nec fermentum urna congue.</p>
+        </article>
+      </section>
+      
+    </div> 
+    <script>
+        $("a.stack").on("click", function(){
```
Comment (donjude-eng) [inline Tutorial12.4/example-3/grid-vs-stack.html:58]: Did you understand this?

File: Tutorial12.4/example-4/style.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    
+}
+.two{
+    background:blue;
+     
+
+}
+.three{
+    background:green;
+
```
Comment (donjude-eng) [inline Tutorial12.4/example-4/style.css]: remove empty spaces?

File: Tutorial12.4/example-2/style.css [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+@media screen and (min-width: 768px){
+
+nav ul{
+  display: flex;
+
+}
+
+nav li{
+  flex: 1 1 0;
+
```
Comment (donjude-eng) [inline Tutorial12.4/example-2/style.css]: remove empty spaces.

**Review by donjude-eng — CHANGES_REQUESTED** [review]

File: Tutorial12.4/example-1/menu.html (hunk 1 above)
Comment (Anisha-Alexander) [reply Tutorial12.4/example-1/menu.html:9]: Will install .

File: Tutorial12.4/example-3/grid-vs-stack.html (hunk 2 above)
Comment (Anisha-Alexander) [reply Tutorial12.4/example-3/grid-vs-stack.html:58]: yes I understood.

File: Tutorial12.4/example-2/style.css (hunk 4 above)
Comment (Anisha-Alexander) [reply Tutorial12.4/example-2/style.css]: done

**Pull request merged**